Assets/MixedRealityToolkit/InputModule/Scripts/Utilities/MicrophoneHelper.cs

```diff
@@ -4,7 +4,7 @@
 //*********************************************************
 //
 // This sample code is a modified version of an official UWP sample which can be found on:
-// https://github.com/Microsoft/Windows-universal-samples/blob/master/Samples/SpeechRecognitionAndSynthesis/cs/AudioCapturePermissions.cs
+// https://github.com/Microsoft/Windows-universal-samples/blob/Dev_Working_Branch/Samples/SpeechRecognitionAndSynthesis/cs/AudioCapturePermissions.cs
```

Comment:
This is a link to an different repo, so it should remain `master`.

Reply:
Darn it. I'll fix that in another PR.  Good spotting (5 mins after merge :O)

Reply:
Yeah...apologies for being slow on the draw!

Reply:
Resolved in #1648